deployment/nether-deploy-quickstart.json

```diff
@@ -1,93 +1,256 @@
-    * For more information, please refer to the Nether repository at: https://github.com/MicrosoftDX/nether/tree/master/deployment
-    */
-    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
-    "contentVersion": "1.0.0.0",
-    "parameters": {
-        "initialNetherAdministratorPassword": {
-            "type": "securestring",
-            "metadata": {
-                "description": "The initial password for the netheradmin user, sql server user, ..."
-            }
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "initialNetherAdministratorPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The initial password for the netheradmin user, sql server user, ..."
+      }
+    },
+    "templateBaseURL": {
+      "type": "string",
+      "defaultValue": "https://raw.githubusercontent.com/MicrosoftDX/nether/master/deployment/",
+      "metadata": {
+        "description": "The base location for all linked templates"
+      }
+    },
+    "WebHostingPlan": {
+      "type": "string",
+      "defaultValue": "Free (no 'always on')",
+      "allowedValues": [
+        "Free (no 'always on')",
+        "Shared",
+        "Basic B1",
+        "Basic B2",
+        "Basic B3",
+        "Standard S1",
+        "Standard S2",
+        "Standard S3",
+        "Premium P1",
+        "Premium P2",
+        "Premium P3"
+      ],
+      "metadata": {
+        "description": "Specifies the database pricing/performance."
+      }
+    }
+  },
+  "variables": {
+    "deploymentAPI": "2015-01-01",
+    "suffix": "[uniqueString(subscription().subscriptionId, resourceGroup().id)]",
+    "netherTemplateURI": "[uri(parameters('templateBaseURL'), 'nether-deploy.json')]",
+    "NetherWebDomainPrefix": "[concat('netherweb', variables('suffix'))]",
+    "sqlServerName": "[concat('nethersql', variables('suffix'))]",
+    "sqlAdministratorLogin": "nethersqladmin",
+    "sqlAdministratorPassword": "[parameters('initialNetherAdministratorPassword')]",
+    "analyticsEventHubNamespace": "[concat('nether', variables('suffix'))]",
+    "InstanceCount": 1,
+    "ManagePolicyName": "managePolicy",
+    "namespaceName": "[concat('nether', uniquestring(resourceGroup().id))]",
+    "ingestEventHub": "[concat('ingest', uniquestring(resourceGroup().id))]",
+    "intermediateEventHub": "[concat('intermidiate', uniquestring(resourceGroup().id))]",
+    "defaultSASKeyName": "RootManageSharedAccessKey",
+    "authRuleResourceId": "[resourceId('Microsoft.EventHub/namespaces/authorizationRules', variables('namespaceName'), variables('defaultSASKeyName'))]",
+    "ehApiVersion": "2015-08-01",
+    "WebResourceAPI": "2015-08-01",
+    "hostingPlanName": "[concat('hostingPlan', uniquestring(resourceGroup().id))]",
+    "webSiteName": "[concat('site', uniquestring(resourceGroup().id))]",
+    "HostingSKUs": {
```

Comment:
I don't see why we are defining HostingSKUs, I can't find where they are used again later in the template